lib/input/mbox/keyword_digest.py

```diff
@@ -21,7 +21,7 @@ def get_top_authors(top_n, json_filename):
     """
     top_authors = set()
     top_authors_index = dict()
-    author_scores = generate_author_ranking(json_filename, output_filename=None, active_score=2, passive_score=1, write_to_file=False)
+    author_scores = get(json_filename, output_filename=None, active_score=2, passive_score=1, write_to_file=False)

```

Comment:
I suggest using `from lib.analysis.author import ranking` and then `ranking.get` as many other libraries will also have `get` function making the code less readable and increasing the possibility for future errors